test/parallel/test-cluster-dgram-1.js

```diff
@@ -111,5 +111,5 @@ function worker() {
     }
   }, PACKETS_PER_WORKER));
 
-  socket.bind(common.PORT);
+  socket.bind(0);
```

Comment:
I don't think this is right. I think each of the four workers will get a different port. In the current test, they are all using the same port.